### PR TITLE
Workarea

### PR DESCRIPTION
**Pull request by atenaiis**
The aim of this project is to build with a responsive design using @media.

I worked alone for this project with Greg Barrett authorization because I did not get paired

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: assets/things/main.css [hunk 1]
```diff
@@ -0,0 +1,886 @@
...
+}
+
+.dropdown-content a:hover {
+  color: white;
+}
+
+.nav-bar-list1 > li > a:hover,
+.nav-bar-list2 > li > a:hover {
+  color: white;
+}
```
Comment (andresporras3423) [inline assets/things/main.css:86]: Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.

File: assets/things/main.css [hunk 2]
```diff
@@ -0,0 +1,886 @@
...
+.white-nav-items2 > li {
+  list-style: none;
+  display: inline-block;
+  padding-right: 10px;
+  padding-top: 15px;
+}
+
+.white-nav-items > li > a:hover {
+  color: black;
+}
```
Comment (andresporras3423) [inline assets/things/main.css:146]: Same style definition as the selector of line 285. Group selectors to minimize the code.

File: assets/things/main.css [hunk 3]
```diff
@@ -0,0 +1,886 @@
...
+
+.bottom-list-items > li::before {
+  content: "•";
+  padding-right: 8px;
+  color: #f42;
+}
+
+.bottom-list-items > li > a:hover {
+  color: #333;
+}
```
Comment (andresporras3423) [inline assets/things/main.css:433]: Same style definition as the selector of line 453. Group selectors to minimize the code.

**Comment by atenaiis** [comment]
thank you for the review,
I have worked on the things that needed to be fixed.
Thank you

**Comment by atenaiis** [comment]
thank you for the review
for the small screen I have change from 375 to 450 so it can be seen.
and fixed the other issues 
ran npx command to fix issues

Cheers!

**Comment by atenaiis** [comment]
thank you for your review.

I have worked on changes requested and kindly ask for another review.

cheers.

Mariana Campos

**Comment by atenaiis** [comment]
worked on last review and fixed issue.

thanks for the review.

cheers!!!!

**Comment by atenaiis** [comment]
thanks for the quick review.

I have implemented the changes.

cheers.

**Comment by sumancrest0001** [comment]
**Status: changes required
Great job so far @atenaiis.** It looks almost similar to the original site on mobile, tablet, and desktop versions. You have ticked all the requirements but some minor issues are still there to be fixed. Please consider the following points.

**Note: all the changes are mandatory.**
1. There is an issue.
Currently :point_down: 
Screenshot from 2020-07-03 10-31-32

it should be :point_down: 
Screenshot from 2020-07-03 10-53-16

2. Let's give some margins to the titles
currently :point_down: 
Screenshot from 2020-07-03 10-40-09

On the original page :point_down: 
Screenshot from 2020-07-03 10-40-01

**Comment by atenaiis** [comment]
thanks for the review,

I have placed the tittles.

cheers!!!

Mariana Campos

**Pull request merged**